### PR TITLE
fix: mzxml

### PR DESCRIPTION
**Pull request by josoriom**

### CODE REVIEW

File: src/mzxml/parseBinaryDataArray.js [hunk 1]
```diff
@@ -6,32 +8,31 @@ import { parseCvParam } from './parseCvParam';
...
-    kind = cvParam.contentType;
-  } else if (cvParam.pairOrder) {
-    kind = cvParam.pairOrder;
-  } else {
-    throw new Error('unknown binary data type');
-  }
-
-  // console.log(buffer)
+  let kind = cvParam.contentType ? cvParam.contentType : cvParam.pairOrder;
+  let buffer = Buffer.from(bytes);
```
Comment (targos) [inline src/mzxml/parseBinaryDataArray.js:22]: Please use a typed array or a data view, not a buffer

File: src/__tests__/mzXML.test.js [hunk 2]
```diff
@@ -6,59 +6,45 @@ import { parseMZ } from '..';
...
-  it('tiny2.0.mzXML', () => {
+  it('read 32 bits mzXML test file', () => {
     const data = readFileSync(join(pathFiles, 'tiny2.0.mzXML'));
     let response = parseMZ(data);
-    expect(response.times).toStrictEqual([1209, 2577]);
+    expect(response.times).toStrictEqual([353.43, 356.68]);
     expect(response.series.ms.data).toHaveLength(2);
-    expect(response.series.ms.data[0][0]).toHaveLength(5252);
-    expect(response.series.ms.data[1][0]).toHaveLength(172);
+    expect(response.series.ms.data[0][0]).toHaveLength(1313);
```
Comment (lpatiny) [inline src/__tests__/mzXML.test.js:14]: Please add a test on the first ms data point mz and intensity

File: src/__tests__/mzXML.test.js [hunk 3]
```diff
@@ -6,59 +6,45 @@ import { parseMZ } from '..';
...
-    expect(response.series.ms.info[1]).toStrictEqual(metadata);
-  });
-
-  it('read bigTest.mzML', () => {
+  it('read 64 bits mzXML test file', () => {
     const data = readFileSync(join(pathFiles, 'bigTest.mzXML'));
     let response = parseMZ(data);
+    let scans = response.series.ms.data;
+    let intensity = scans[0][0].reduce((a, b) => a + b);
+    let mz = scans[0][1].reduce((a, b) => a + b);
```
Comment (lpatiny) [inline src/__tests__/mzXML.test.js:35]: Please also add a test on the first ms data point mz and intensity

File: src/mzxml/parseBinaryDataArray.js [hunk 4]
```diff
@@ -6,32 +8,31 @@ import { parseCvParam } from './parseCvParam';
 export function parseBinaryDataArray(node) {
   let data = node.peaks;
   let attr = node._attr;
+  let string = data._data;
+  let base64 = /^([A-Za-z0-9/+]{4})*(([A-Za-z0-9/+]{3}=)|([A-Za-z0-9/+]{2}==))?$/;
```
Comment (lpatiny) [inline src/mzxml/parseBinaryDataArray.js:12]: When possible you may define `const` rather that `let`. A habit to take.

File: src/mzxml/parseBinaryDataArray.js (hunk 1 above)
Comment (lpatiny) [reply src/mzxml/parseBinaryDataArray.js:22]: I guess this applies to the whole project. @josoriom could you have a look and search globally for Buffer ?

**Pull request merged**